src/modules/users/components/TokenActivation/StakesTab/StakesTab.tsx

```diff
@@ -0,0 +1,91 @@
+import React from 'react';
+import { FormattedMessage, defineMessages } from 'react-intl';
+
+import { Address } from '~types/index';
+import { SpinnerLoader } from '~core/Preloaders';
+import { ParsedMotionStakedEvent, UserToken } from '~data/generated';
+
+import { getFormattedTokenValue } from '~utils/tokens';
+import { FullColonyFragment } from '~data/index';
+
+import ClaimAllButton from './ClaimAllButton';
+import StakesListItem from './StakesListItem';
+
+import styles from './StakesTab.css';
+
+const MSG = defineMessages({
+  yourStakes: {
+    id: 'users.TokenActivation.TokenActivationContent.StakesTab.yourStakes',
```

Comment:
Same here, it's `TokenActivation` instead of `TokenActivationContent` right?

Reply:
Yep - 100%. I refactored into another dir and didnt update the ids. I'll add the update.